From 64ee29ad8a94480af037f35ade04810ca0e1031d Mon Sep 17 00:00:00 2001
From: aleth <aleth@instantbird.org>
Date: Wed, 26 Oct 2016 20:16:58 +0200
Subject: [PATCH 22/28] Bug 1313137 - "msg is not defined" error in
 irc.js:changeBuddyNick. r=clokep

--HG--
extra : rebase_source : 5752a69059ecd48b947809ef12de177ccab8528f
---
 chat/protocols/irc/irc.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chat/protocols/irc/irc.js b/chat/protocols/irc/irc.js
index c2167a5ec..b58c2c2eb 100644
--- a/chat/protocols/irc/irc.js
+++ b/chat/protocols/irc/irc.js
@@ -1289,7 +1289,8 @@ ircAccount.prototype = {
       this.conversations.set(aNewNick, conversation);
 
       conversation.updateNick(aNewNick);
-      conversation.writeMessage(aOldNick, msg, {system: true});
+      conversation.writeMessage(aOldNick, _conv("nickSet", aOldNick, aNewNick),
+                                {system: true});
     }
   },
 
-- 
2.11.0

